Acrolinx 1/15/20 (#16700)
dotnet/docs: docs/csharp/expression-trees-translating.md

ms.assetid: b453c591-acc6-4e08-8175-97e5bc65958e
---

> # Translate Expression Trees

Reviewer: ```suggestion
# Translate expression trees
```

modified. This can result in significant memory efficiencies.
The same nodes can be used throughout a tree, or in multiple
expression trees. Since nodes can't be modified, the

> same node can be reused whenever it is needed.

Reviewer: ```suggestion
same node can be reused whenever it's needed.
```
